tests/test_cr_detection.py

```diff
@@ -799,6 +800,9 @@ def test_p_below_min():
     if not i_beg is None:
         raise Exception("find_plateaus did not ignore plateau below sat threshold.")
 
+def test_remove_sat_images():
+    ds = Dataset(["oversat_image.fits"])
```

Comment:
instead of a fits file, can we just generate a fake image on the fly with a few saturated values? See e.g., the unit tests in test_combine.py